Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -383,12 +383,17 @@ public function findBlockEnd($type, $searchIndex, $findEnd = true)
      * Find tokens of given kind.
      *
      * @param int|array $possibleKind kind or array of kind
+     * @param int|null  $start        optional offset
```

Comment:
Also, I think we should ALWAYS use null as a default, like symfony and php do, so you can skip default values with a null as a convention.